packages/mpv/build.sh

```diff
@@ -6,7 +6,8 @@ TERMUX_PKG_MAINTAINER="Joshua Kahn @TomJo2000"
 TERMUX_PKG_VERSION="0.40.0"
 TERMUX_PKG_SRCURL=https://github.com/mpv-player/mpv/archive/v${TERMUX_PKG_VERSION}.tar.gz
 TERMUX_PKG_SHA256=10a0f4654f62140a6dd4d380dcf0bbdbdcf6e697556863dc499c296182f081a3
-TERMUX_PKG_AUTO_UPDATE=false
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_AUTO_UPDATE_GROUP=x11/mpv
```

Comment:
Right, so how did you choose the names for the update groups?
Because It's obviously not based on the file paths.
And I'd prefer to not bring channel information into the group names.

If the update group names are "arbitrary" I'd prefer them to follow the project name or a suitable proxy like `kf6`
```suggestion
TERMUX_PKG_AUTO_UPDATE_GROUP="mpv"
```

Reply:
Name of update group is copied directly to PR description and, as a consequence, to git commit in the case if you squash two commits after PR reviewing.

Reply:
There may also be a case where a package may be part of multiple autoupdate groups.
Although that is probably best left for when it actually happens.

Reply:
> In which case `bump(x11/kframeworks6): group update` vs. `bump(kf6): group update` is a purely cosmetic difference in the PR title.

All packages under `kf6` group have version aligning enabled so commit message will be, i.e., `bump(x11/kf6): 6.14.0` which complies with commit messages we had before.
Currently all other update groups or have single package (big package) or have version aligning.

> There may also be a case where a package may be part of multiple autoupdate groups.

It should not happen. It is not designed this way.

Reply:
> > In which case `bump(x11/kframeworks6): group update` vs. `bump(kf6): group update` is a purely cosmetic difference in the PR title.
> 
> All packages under `kf6` group have version aligning enabled so commit message will be, i.e., `bump(x11/kf6): 6.14.0` which complies with commit messages we had before. Currently all other update groups or have single package (big package) or have version aligning.

I don't think the versions always matching or all packages in a group always being updated at the same time is a particularly good assumption to make.
Say I'd like to make an update group for `tree-sitter` later for example,
those versions don't match, and they're often but not always updated at the same time.
Although I suppose I shouldn't be conflating this with a "rebuilds group".

> > There may also be a case where a package may be part of multiple autoupdate groups.
> 
> It should not happen. It is not designed this way.

*Not right now.*
But yeah no point implementing a feature we don't have a need for yet.

Reply:
In my understanding auto-update group definition is similar to rebuild group: group of packages which are required to be updated in the same PR/push because of version of ABI mismatch. I planned to implement something like creating autoupdate group for, i.e. boost with automatic revbumping of all revdeps (because of possible ABI mismatch of newer boost version). 

> But yeah no point implementing a feature we don't have a need for yet.

As I said before my understanding of update group is similar to rebuild group so I do not see the point.

Reply:
> ```suggestion
> TERMUX_PKG_AUTO_UPDATE_GROUP="mpv"
> ```

Since these two packages exist in different repos we can omit the `x11/` preffix. But PR title will be less fine after this :disappointed:  .

Reply:
I'm gonna reiterate that I don't think repo channel information should be part of the update group names.

Reply:
Probably it should be in the case if all packages in update group exist in the same repo. Just for consistency with other `bump` commits.